basic-javascript/project-1.spec.js

```diff
@@ -7,71 +7,108 @@ describe('project-1', () => {
     it('returns NaN when given a non numeric value', () => {
       expect(helpers.multiplyByTen(undefined)).toBeNaN();
       expect(helpers.multiplyByTen('two')).toBeNaN();
-    });
-
+    });    
     it('should return the given number multiplied by 10', () => {});
+    expect(helpers.multiplyByTen(2)).toBe(20);
   });
 });
+describe('project-1', () => {
+  describe('subtractFive', () => {
+    it('returns NaN when given a non numeric value', () => {
+        expect(helpers.subtractFive(undefined)).toBeNaN();
+        expect(helpers.subtractFive('two')).toBeNaN();
+    });
 
-// describe('subtractFive', () => {
-//    it(, () => {
-
-//    })
-// })
-
-// describe('areSameLength', () => {
-//     it(, () => {
- 
-//     })
-//  })
-
-//  describe('areEqual', () => {
-//     it(, () => {
- 
-//     })
-//  })
+    it('should return the given number substracted by 5', () => {
+        expect(helpers.subtractFive(30)).toBe(25)
+   });
+ });
+});
 
-//  describe('lessThanNinety', () => {
-//     it(, () => {
+describe('project-1', () => {
+  describe('areSameLength', () => {
+    it('should return a boolean to indicate equal comparison', () => {
+        expect(helpers.areSameLength('this', 'that')).toBe(true);       
+    });
+  });
+});
+describe('project-1', () => {
+  describe('areEqual', () => {
+    it('should return a boolean to indicate equalility', () => {
+        expect(helpers.areEqual(5, 5)).toBe(true);       
+      }); 
+    });
+ });
+
+ describe('project-1', () => {
+   describe('lessThanNinety', () => {
+    it('should return a boolean to indicate a given number is less than 90', () => {
+        expect(helpers.lessThanNinety(70)).toBe(true);      
+        expect(helpers.lessThanNinety(100)).toBe(false);         
+      }); 
+   });
+ });
+
+ describe('project-1', () => {
+  describe('greaterThanFifty', () => {
+    it('should return a boolean to indicate a given number is greater than 50', () => {
+        expect(helpers.greaterThanFifty(70)).toBe(true);  
+        expect(helpers.greaterThanFifty(20)).toBe(false); 
  
```

Comment:
a nice edge case would be: test 50 and 51 also, so the developer did not use >= instead of >